content/hi/version_control.md

```diff
@@ -0,0 +1,17 @@
+---
+title: वर्शन कण्ट्रोल (Version Control)
```

Comment:
Kindly remove the english part.

Reply:
> Kindly remove the english part.
You mean (Version Control) should be removed?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Not reusable